cognite/client/_api/datapoints.py

```diff
@@ -536,13 +526,9 @@ def __init__(self, config: ClientConfig, api_version: str | None, cognite_client
     def retrieve(
         self,
         *,
-        id: None | int | DatapointsQuery | dict[str, Any] | Sequence[int | DatapointsQuery | dict[str, Any]] = None,
-        external_id: None
-        | str
-        | DatapointsQuery
-        | dict[str, Any]
-        | SequenceNotStr[str | DatapointsQuery | dict[str, Any]] = None,
-        instance_id: None | NodeId | Sequence[NodeId] = None,
+        id: None | int | DatapointsQuery | Sequence[int | DatapointsQuery] = None,
```

Comment:
isn't this breaking? not to allow `dict[str, Any]` any more?

Reply:
Just the type annotations being updated to push people a little bit 😉